src/IB/IIMethod.cpp

```diff
@@ -478,6 +483,11 @@ IIMethod::preprocessIntegrateData(double current_time, double new_time, int /*nu
         d_U_half_vecs[part] = dynamic_cast<PetscVector<double>*>(
             d_U_current_vecs[part]->clone().release()); // WARNING: must be manually deleted
 
+        d_U_old_systems[part] = &d_equation_systems[part]->get_system(VELOCITY_OLD_SYSTEM_NAME);
+        d_U_old_vecs[part] = dynamic_cast<PetscVector<double>*>(d_U_old_systems[part]->current_local_solution.get());
+        d_U_old_updated_vecs[part] = dynamic_cast<PetscVector<double>*>(
+            d_U_old_vecs[part]->clone().release()); // WARNING: must be manually deleted
```

Comment:
Same as above.

Reply:
Hello Boyce, 
I am wondering should I keep the U_old_systems for now and change the solver to only configured when we do multistep time stepping? Or also remove the system and try to store vector in other way?

Reply:
You can keep `U_old_system` for now --- I had forgotten that `IBFEMethod` was using `LibMeshSystemVector` to handle DOF vectors, which is how it avoids needing to have the extra `System`.

I think we could clean up the implementation if we switched to using `LibMeshSystemVector` to manage the different DOF vectors, as is done in `IBFEMethod`. Could you possibly work on that in a separate PR?

Reply:
Sure. Let me clean up this PR. I can take a look and try to add LibMeshSystemVector.